src/iterative_sorting/iterative_sorting.py

```diff
@@ -5,22 +5,30 @@ def selection_sort( arr ):
         cur_index = i
         smallest_index = cur_index
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
-
-
+        # (hint, can do in 3 loc)
+        for j in range(cur_index, len(arr)):
+            if arr[j] < arr[smallest_index]:
+                smallest_index = j
         # TO-DO: swap
-
-
-
+        temp = arr[smallest_index]
+        arr[smallest_index] = arr[cur_index]
+        arr[cur_index] = temp
 
     return arr
 
 
 # TO-DO:  implement the Bubble Sort function below
 def bubble_sort( arr ):
-
+    did_swap = True
+    while did_swap:
+        did_swap = False
+        for i in range(0, len(arr)-1):
+            if arr[i] > arr[i+1]:
+                # swap
+                temp = arr[i]
+                arr[i] = arr[i+1]
+                arr[i+1] = temp
+                did_swap = True
```

Comment:
Good use of a while loop and a boolean instead of a count!
Python has a nice feature for swapping
```
arr[i], arr[i+1] = arr[i+1], arr[i]
```
Is equivalent to the swapping above